blackhole/database_utils.py

```diff
@@ -242,15 +242,23 @@ def retrieve_takes_by_list(slate_and_take_list: TakeIDsList, include_corrections
         cursor = dbConnection.cursor()
 
         try:
-            list_as_sql_string = ", ".join([f'("{t[0]}", {t[1]})' for t in slate_and_take_list.id_list])
+            id_list = slate_and_take_list.id_list
 
-            command = (f"SELECT * FROM blackhole_takes WHERE ({SLATE_DB_COL}, {TAKE_NUMBER_DB_COL}) IN "
-                       f"({list_as_sql_string})")
+            sql = f"SELECT * FROM blackhole_takes WHERE (:slate_col, :take_col) IN :id_list"
+            args = {
+                "slate_col": SLATE_DB_COL,
+                "take_col": TAKE_NUMBER_DB_COL,
+                "id_list": id_list
+            }
 
             if include_corrections:
-                command += f" OR ({CORRECTED_SLATE_DB_COL}, {CORRECTED_TAKE_DB_COL}) IN ({list_as_sql_string})"
+                sql += f" OR (:corrected_slate_col, :corrected_take_col) in :id_list"
```

Comment:
I should add that this feels like this expression could be simplified from `(A, B) in X | (C, D) in X` to `(A, B) | (C, D) in X` but I don't trust my very rusty SQL especially as there isn't a test suite for these functions (and there probably should be).